roles/jenkins_builder/tasks/main.yml

```diff
@@ -13,6 +13,7 @@
 
 - include: setup_disk_{{ location }}.yml
   when: ansible_system == 'Linux' and location
+  tags: key
```

Comment:
We do not really use tag, I might remove it on merge.

Reply:
Yeah, this was only there for testing. Slipped through. I've removed it already.